spec/api-browser-window-spec.js

```diff
@@ -2532,12 +2532,8 @@ describe('BrowserWindow module', () => {
 
     it('should save page to disk', async () => {
       await w.loadFile(path.join(fixtures, 'pages', 'save_page', 'index.html'))
-      const error = await new Promise(resolve => {
-        w.webContents.savePage(savePageHtmlPath, 'HTMLComplete', function (error) {
-          resolve(error)
-        })
-      })
-      expect(error).to.be.null()
```

Comment:
Shouldn't we keep the assert that error is null, eg change the await to contain a try/catch or use .then().catch()

Reply:
In promise-land the only place to assert this error expectation would be in something like `.catch(err)`, but a catch function would only ever be triggered if there _is_ an error..so i'm fairly certain that's it'd be a catch-22 codepath in that it would never be asserted 99% of the time and the other 1% of the time it would fail by necessity of the conditions necessary to reach that assertion in the first place 🤔

Reply:
ok I think I'm fine with that as there are other asserts to verify savePage worked.